### ISSUE TITLE
SSL: Certificate Failure Episode Async web scraping

### ISSUE THREAD

**Issue by androiddrew** [talkpython/async-techniques-python-course#7]
For the web scraping component against https://talkpython.fm I was successful using the sync libraries and code with a 3.6 interpreter, however when moving on to the async version I get:
```
aiohttp.client_exceptions.ClientConnectorSSLError: Cannot connect to host talkpython.fm:443 ssl:None [[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:833)]
```

**Comment by mikeckennedy** [comment]
I started running into this issue after awhile too. Was very weird. Must be some update to aiohttp? Anyway, you can avoid the error if you set a value in the `ClientSession`, see this example:

https://github.com/talkpython/async-techniques-python-course/blob/23d7f68b5289e3478b70b30fd206f3dab42996e7/src/09-built-on-asyncio/the_unsync/presync.py

**Comment by androiddrew** [comment]
So the 3.6.5 interpreter I am using was installed from python.org. When I homebrew installed python I got a 3.7 interpreter and I did notice that openssl was present under the Cellar. 

Checking the 3.7 interpreter I got:
```
>>> import ssl
>>> ssl.OPENSSL_VERSION
'OpenSSL 1.0.2p  14 Aug 2018'
 ```

The 3.6.5 interpreter had:
```
>>> import ssl
>>> ssl.OPENSSL_VERSION
'OpenSSL 1.0.2n  7 Dec 2017'
```

So the issue could be this maybe.  I am switching to the homebrew version for the rest of the class I guess.
